crates/nu-plugin/src/protocol/plugin_custom_value/tests.rs

```diff
@@ -265,145 +270,6 @@ fn verify_source_error_message() -> Result<(), ShellError> {
     Ok(())
 }
 
-#[test]
-fn verify_source_nested_range() -> Result<(), ShellError> {
-    let native_val = Value::test_custom_value(Box::new(TestCustomValue(32)));
-    let source = PluginSource::new_fake("test");
-    for (name, mut val) in [
-        (
-            "from",
```

Comment:
Are those tests obsolete, did not see them again in my scrolling.

Reply:
`verify_source` now only operates on a single value without recursion, because I moved the recursion further up. The equivalent test is now located with `with_custom_values_in()`